.github/CHANGELOG.md

```diff
@@ -13,11 +13,14 @@
 * Update Lightning tag to latest_release.
 [(#51)](https://github.com/PennyLaneAI/pennylane-lightning-gpu/pull/51)
 
+* Reintroduce dispatching support for SingleExcitation and DoubleExcitation gates in C++ layer.
```

Comment:
I'm not sure if we still use this code formatting within changelog :smile: 
```suggestion
* Reintroduce dispatching support for `SingleExcitation` and `DoubleExcitation` gates in C++ layer.
```

Reply:
Good catch!